Release notes updates for CC 18.3 (#1282)
tigera/docs: calico-cloud_versioned_docs/version-18-2/release-notes/index.mdx

## January 31, 2024 (version 18.3.0)

### New features and enhancements

#### Assign custom roles to users automatically with Entra ID (formerly Azure AD) groups

We've added the ability to link custom roles in Calico Cloud to your organization's Entra ID groups.
You can define and modify group membership in Entra ID, and Calico Cloud will automatically grant role-based access to users based on that group membership.

For more information, see [Create a custom role for an Entra ID group](../users/create-and-assign-custom-roles.mdx))

#### Export custom roles

> In this release you can export custom roles from a managed cluster and apply them to another managed cluster.

Reviewer: ```suggestion
We've added the ability to export custom roles from a managed cluster and apply them to another managed cluster.
```
